roachtest: backupTPCC failed

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656809&tab=buildLog) on [release-2.1@b02795e4629ec045d3bfb5f72e8695166863f199](https://github.com/cockroachdb/cockroach/commits/b02795e4629ec045d3bfb5f72e8695166863f199):

```
The test failed on branch=release-2.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191220-1656809/backupTPCC/run_1
	cluster.go:2166,backup.go:132,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: nodelocal does not support hosts: nodelocal://1/teamcity-1656809-1576825466-15-n3cpu4/full
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656809&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656791&tab=buildLog) on [release-19.2@a84266e50207e2e9a27828991ec9fc2f62d74a88](https://github.com/cockroachdb/cockroach/commits/a84266e50207e2e9a27828991ec9fc2f62d74a88):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191220-1656791/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 57 ranges: importing span /Table/53/{1-2}: fetching "513667183854649347.sst": open /mnt/data1/cockroach/extern/teamcity-1656791-1576827721-17-n3cpu4/full/513667183854649347.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1656791&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658803&tab=buildLog) on [release-19.1@a85a9a4ee0440cccdfd3e314b8f01394e52bdf3f](https://github.com/cockroachdb/cockroach/commits/a85a9a4ee0440cccdfd3e314b8f01394e52bdf3f):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191221-1658803/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 60 ranges: importing span /Table/53/{1-2}: fetching "513940891985379329.sst": open /mnt/data1/cockroach/extern/teamcity-1658803-1576912021-06-n3cpu4/full/513940891985379329.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658803&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658821&tab=buildLog) on [release-19.2@8da55b5585cc2e3dd3b87b9d01d2000c03ea9953](https://github.com/cockroachdb/cockroach/commits/8da55b5585cc2e3dd3b87b9d01d2000c03ea9953):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191221-1658821/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:700: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 46 ranges: importing span /Table/55/{2-3}: fetching "513949547060887554.sst": open /mnt/data1/cockroach/extern/teamcity-1658821-1576914157-17-n3cpu4/full/513949547060887554.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1658821&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1660313&tab=buildLog) on [master@319080c701cb4e13c347e86c4afb2f1c2af78def](https://github.com/cockroachdb/cockroach/commits/319080c701cb4e13c347e86c4afb2f1c2af78def):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191222-1660313/backupTPCC/run_1
	cluster.go:1786,backup.go:89,test_runner.go:700: error with attached stack trace:
		    main.execCmd
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:399
		    main.(*cluster).RunL
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1820
		    main.(*cluster).Run
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:1784
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:89
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:700
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - error with embedded safe details: %s returned:
		    stderr:
		    %s
		    stdout:
		    %s
		    -- arg 1: <string>
		    -- arg 2: <string>
		    -- arg 3: <string>
		  - /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod run teamcity-1660313-1577000643-17-n3cpu4:1 -- ./workload init tpcc --warehouses=10 {pgurl:1-3} returned:
		    stderr:
		    
		    stdout:
		    I191222 07:56:48.547388 1 workload/workloadsql/dataload.go:135  imported warehouse (0s, 10 rows)
		    I191222 07:56:48.582284 1 workload/workloadsql/dataload.go:135  imported district (0s, 100 rows)
		    I191222 07:57:03.181593 1 workload/workloadsql/dataload.go:135  imported customer (15s, 300000 rows)
		    I191222 07:57:11.844438 1 workload/workloadsql/dataload.go:135  imported history (9s, 300000 rows)
		    I191222 07:57:17.867282 1 workload/workloadsql/dataload.go:135  imported order (6s, 300000 rows)
		    I191222 07:57:18.454425 1 workload/workloadsql/dataload.go:135  imported new_order (1s, 90000 rows)
		    I191222 07:57:19.648581 1 workload/workloadsql/dataload.go:135  imported item (1s, 100000 rows)
		    I191222 07:58:03.899346 1 workload/workloadsql/dataload.go:135  imported stock (44s, 1000000 rows)
		    Error: failed insert into order_line: dial tcp 10.128.0.120:26257: connect: connection refused
		    Error:  exit status 1:
		  - exit status 1
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1660313&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661413&tab=buildLog) on [release-19.2@d23a3df4b9eddd07f02b67b6beb16aee8fd8e65b](https://github.com/cockroachdb/cockroach/commits/d23a3df4b9eddd07f02b67b6beb16aee8fd8e65b):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191224-1661413/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 54 ranges: importing span /Table/53/{1-2}: fetching "514798174350639105.sst": open /mnt/data1/cockroach/extern/teamcity-1661413-1577173177-17-n3cpu4/full/514798174350639105.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1661413&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662289&tab=buildLog) on [release-19.2@b4efe661f4bff33554fe41bd4a1a5f245a6fde2b](https://github.com/cockroachdb/cockroach/commits/b4efe661f4bff33554fe41bd4a1a5f245a6fde2b):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191225-1662289/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 50 ranges: importing span /Table/56/{1-2}: fetching "515079855757983747.sst": open /mnt/data1/cockroach/extern/teamcity-1662289-1577259113-17-n3cpu4/full/515079855757983747.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1662289&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664336&tab=buildLog) on [release-19.2@599c988773fa6fb4544369a50fe8d06b90c64171](https://github.com/cockroachdb/cockroach/commits/599c988773fa6fb4544369a50fe8d06b90c64171):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191227-1664336/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 59 ranges: importing span /Table/56/{1-2}: fetching "515647446381920257.sst": open /mnt/data1/cockroach/extern/teamcity-1664336-1577432048-17-n3cpu4/full/515647446381920257.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1664336&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=buildLog) on [release-19.2@8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3](https://github.com/cockroachdb/cockroach/commits/8ca08b3ddbfa07eb8fc19a1035cf82539f05c7a3):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191228-1665474/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 53 ranges: importing span /Table/53/{1-2}: fetching "515929647315484673.sst": open /mnt/data1/cockroach/extern/teamcity-1665474-1577518491-17-n3cpu4/full/515929647315484673.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1665474&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1667196&tab=buildLog) on [release-19.1@cf4d9a46193b9fc2c63b7adf89b3b0b5c84adb2b](https://github.com/cockroachdb/cockroach/commits/cf4d9a46193b9fc2c63b7adf89b3b0b5c84adb2b):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191230-1667196/backupTPCC/run_1
	cluster.go:2166,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2157
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2165
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2210
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 81 ranges: importing span /Table/56/{3-4}: fetching "516489457149804545.sst": open /mnt/data1/cockroach/extern/teamcity-1667196-1577689805-10-n3cpu4/full/516489457149804545.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1667196&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668225&tab=buildLog) on [release-2.1@c96086a9f10c5dc3a79cf1e8e80e4a0dc878b120](https://github.com/cockroachdb/cockroach/commits/c96086a9f10c5dc3a79cf1e8e80e4a0dc878b120):

```
The test failed on branch=release-2.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668225/backupTPCC/run_1
	cluster.go:2189,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: nodelocal does not support hosts: nodelocal://1/teamcity-1668225-1577775329-05-n3cpu4/full
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668225&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=buildLog) on [release-19.2@3960c23e7b8fef3042d525f74768c5ae85a1cf22](https://github.com/cockroachdb/cockroach/commits/3960c23e7b8fef3042d525f74768c5ae85a1cf22):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191231-1668207/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 56 ranges: importing span /Table/53/{1-2}: fetching "516781259481088002.sst": open /mnt/data1/cockroach/extern/teamcity-1668207-1577777755-17-n3cpu4/full/516781259481088002.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1668207&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671751&tab=buildLog) on [release-19.1@5b73cd419b514452492226c0ea1b19639356ebb8](https://github.com/cockroachdb/cockroach/commits/5b73cd419b514452492226c0ea1b19639356ebb8):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671751/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 70 ranges: importing span /Table/53/{1-2}: fetching "517622636843827201.sst": open /mnt/data1/cockroach/extern/teamcity-1671751-1578035659-15-n3cpu4/full/517622636843827201.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671751&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=buildLog) on [release-19.2@9aaf5fb6f7108858d200dd3d199deb11871a09b4](https://github.com/cockroachdb/cockroach/commits/9aaf5fb6f7108858d200dd3d199deb11871a09b4):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200103-1671769/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 58 ranges: importing span /Table/53/{1-2}: fetching "517629822849318914.sst": open /mnt/data1/cockroach/extern/teamcity-1671769-1578037401-17-n3cpu4/full/517629822849318914.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1671769&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=buildLog) on [release-19.2@6cc3423103f85d7f82f01791a24d99bef21b2546](https://github.com/cockroachdb/cockroach/commits/6cc3423103f85d7f82f01791a24d99bef21b2546):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200104-1673640/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 54 ranges: importing span /Table/53/{1-2}: fetching "517910977123844099.sst": open /mnt/data1/cockroach/extern/teamcity-1673640-1578123152-17-n3cpu4/full/517910977123844099.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1673640&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=buildLog) on [release-19.2@f3bcbd305602c89d52cb7ea4cb114f362c5aee23](https://github.com/cockroachdb/cockroach/commits/f3bcbd305602c89d52cb7ea4cb114f362c5aee23):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200107-1676733/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 56 ranges: importing span /Table/53/{1-2}: fetching "518762219501486083.sst": open /mnt/data1/cockroach/extern/teamcity-1676733-1578382909-17-n3cpu4/full/518762219501486083.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1676733&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1678603&tab=buildLog) on [master@873c377178a140d0e997741646b6c6e4318f7e87](https://github.com/cockroachdb/cockroach/commits/873c377178a140d0e997741646b6c6e4318f7e87):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200108-1678603/backupTPCC/run_1
	cluster.go:2189,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - write tcp 172.17.0.2:36546->34.69.230.236:26257
		  - write
		  - broken pipe
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1678603&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1680663&tab=buildLog) on [release-19.1@3d6bb038f3afcf805fb48dddcffa342195d3d7e0](https://github.com/cockroachdb/cockroach/commits/3d6bb038f3afcf805fb48dddcffa342195d3d7e0):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200109-1680663/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 114 ranges: importing span /Table/53/{1-2}: fetching "519321812119519234.sst": open /mnt/data1/cockroach/extern/teamcity-1680663-1578554053-11-n3cpu4/full/519321812119519234.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1680663&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1680645&tab=buildLog) on [master@8d05b3279385d6c21bccb8610039c2678a7e2391](https://github.com/cockroachdb/cockroach/commits/8d05b3279385d6c21bccb8610039c2678a7e2391):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200109-1680645/backupTPCC/run_1
	cluster.go:2189,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 2: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1680645&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683261&tab=buildLog) on [release-19.2@486a8ac25e0470211f6f7659b26c82950ed013be](https://github.com/cockroachdb/cockroach/commits/486a8ac25e0470211f6f7659b26c82950ed013be):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200110-1683261/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 49 ranges: importing span /Table/53/{1-2}: fetching "519611102485184514.sst": open /mnt/data1/cockroach/extern/teamcity-1683261-1578641619-17-n3cpu4/full/519611102485184514.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683261&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683243&tab=buildLog) on [master@413fd1e46c2936029458db495efa242acb1b7d52](https://github.com/cockroachdb/cockroach/commits/413fd1e46c2936029458db495efa242acb1b7d52):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200110-1683243/backupTPCC/run_1
	cluster.go:2189,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - write tcp 172.17.0.2:49980->35.239.213.29:26257
		  - write
		  - broken pipe
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1683243&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1684749&tab=buildLog) on [release-19.2@07057cf68ec19eb0777f611fedaaefde72fe673a](https://github.com/cockroachdb/cockroach/commits/07057cf68ec19eb0777f611fedaaefde72fe673a):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200111-1684749/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 47 ranges: importing span /Table/53/{1-2}: fetching "519894463612813313.sst": open /mnt/data1/cockroach/extern/teamcity-1684749-1578727929-17-n3cpu4/full/519894463612813313.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1684749&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1684731&tab=buildLog) on [master@9a89b2e038d2fc67e9e155742e3e18316bf08d11](https://github.com/cockroachdb/cockroach/commits/9a89b2e038d2fc67e9e155742e3e18316bf08d11):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200111-1684731/backupTPCC/run_1
	cluster.go:2189,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - write tcp 172.17.0.2:51856->35.193.248.55:26257
		  - write
		  - broken pipe
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1684731&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=buildLog) on [release-19.2@7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37](https://github.com/cockroachdb/cockroach/commits/7f2413b36fe1ea31a08c3ddd0d1e4b1cd006db37):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200114-1688037/backupTPCC/run_1
	cluster.go:2189,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2180
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2188
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2233
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 57 ranges: importing span /Table/53/{1-2}: fetching "520742017777369089.sst": open /mnt/data1/cockroach/extern/teamcity-1688037-1578987224-17-n3cpu4/full/520742017777369089.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1688037&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690444&tab=buildLog) on [release-19.1@67be21c787f727df04e77135ad19d134b8e5c762](https://github.com/cockroachdb/cockroach/commits/67be21c787f727df04e77135ad19d134b8e5c762):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690444/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 84 ranges: importing span /Table/56/{3-4}: fetching "521018349170425857.sst": open /mnt/data1/cockroach/extern/teamcity-1690444-1579071920-11-n3cpu4/full/521018349170425857.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690444&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=buildLog) on [release-19.2@99a4ab5a34b310338098a6aafb6621920fd9d4cc](https://github.com/cockroachdb/cockroach/commits/99a4ab5a34b310338098a6aafb6621920fd9d4cc):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200115-1690462/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 48 ranges: importing span /Table/53/{1-2}: fetching "521026630706266114.sst": open /mnt/data1/cockroach/extern/teamcity-1690462-1579073899-17-n3cpu4/full/521026630706266114.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1690462&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=buildLog) on [release-19.2@297c398803afd80276db8dc78b3f7294822c87a8](https://github.com/cockroachdb/cockroach/commits/297c398803afd80276db8dc78b3f7294822c87a8):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200116-1693632/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 63 ranges: importing span /Table/53/{1-2}: fetching "521308978230067201.sst": open /mnt/data1/cockroach/extern/teamcity-1693632-1579160360-12-n3cpu4/full/521308978230067201.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1693632&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695931&tab=buildLog) on [release-19.1@265263f40589020c2c6505e102a3c5d398054377](https://github.com/cockroachdb/cockroach/commits/265263f40589020c2c6505e102a3c5d398054377):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200117-1695931/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 47 ranges: importing span /Table/55/1{-/2/6/2126}: fetching "521584363615748097.sst": open /mnt/data1/cockroach/extern/teamcity-1695931-1579244662-16-n3cpu4/full/521584363615748097.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695931&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695949&tab=buildLog) on [release-19.2@e00e840cad174e32112fac28e8b2264723997759](https://github.com/cockroachdb/cockroach/commits/e00e840cad174e32112fac28e8b2264723997759):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200117-1695949/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 54 ranges: importing span /Table/53/{1-2}: fetching "521590704482648067.sst": open /mnt/data1/cockroach/extern/teamcity-1695949-1579246502-16-n3cpu4/full/521590704482648067.sst: no such file or directory
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695949&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695913&tab=buildLog) on [master@96d7e50acfaa63b5b60addd7cec8e750a4a4d333](https://github.com/cockroachdb/cockroach/commits/96d7e50acfaa63b5b60addd7cec8e750a4a4d333):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200117-1695913/backupTPCC/run_1
	cluster.go:2201,backup.go:132,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:132
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - unexpected node event: 2: dead
```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1695913&tab=artifacts#/backupTPCC)
```
make stressrace TESTS=backupTPCC PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1698230&tab=buildLog) on [release-19.1@a8444e28d3e7bf07c8ee0c6d5809e7466b57f6f9](https://github.com/cockroachdb/cockroach/commits/a8444e28d3e7bf07c8ee0c6d5809e7466b57f6f9):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200118-1698230/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 78 ranges: importing span /Table/53/{1-2}: fetching "521868064761774082.sst": open /mnt/data1/cockroach/extern/teamcity-1698230-1579331271-16-n3cpu4/full/521868064761774082.sst: no such file or directory

```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1698230&tab=artifacts#/backupTPCC)
[roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2AbackupTPCC.%2A&sort=title&restgroup=false&display=lastcommented+project)

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1698248&tab=buildLog) on [release-19.2@4b12dcfe61b6a7fb7fae5ab50208753cd3e08f46](https://github.com/cockroachdb/cockroach/commits/4b12dcfe61b6a7fb7fae5ab50208753cd3e08f46):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200118-1698248/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 54 ranges: importing span /Table/53/{1-2}: fetching "521874906274496513.sst": open /mnt/data1/cockroach/extern/teamcity-1698248-1579333204-16-n3cpu4/full/521874906274496513.sst: no such file or directory

```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1698248&tab=artifacts#/backupTPCC)
[roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2AbackupTPCC.%2A&sort=title&restgroup=false&display=lastcommented+project)

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1702949&tab=buildLog) on [provisional_202001212141_v19.1.7@a8444e28d3e7bf07c8ee0c6d5809e7466b57f6f9](https://github.com/cockroachdb/cockroach/commits/a8444e28d3e7bf07c8ee0c6d5809e7466b57f6f9):

```
The test failed on branch=provisional_202001212141_v19.1.7, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200122-1702949/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 86 ranges: importing span /Table/53/{1-2}: fetching "522922106078986243.sst": open /mnt/data1/cockroach/extern/teamcity-1702949-1579652818-16-n3cpu4/full/522922106078986243.sst: no such file or directory

```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1702949&tab=artifacts#/backupTPCC)
[roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2AbackupTPCC.%2A&sort=title&restgroup=false&display=lastcommented+project)

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).backupTPCC failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1703424&tab=buildLog) on [release-19.2@4b808c3af1a7318f18d708c89b4d88b9e26377b9](https://github.com/cockroachdb/cockroach/commits/4b808c3af1a7318f18d708c89b4d88b9e26377b9):

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20200122-1703424/backupTPCC/run_1
	cluster.go:2201,backup.go:251,test_runner.go:716: error with attached stack trace:
		    main.(*monitor).WaitE
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2192
		    main.(*monitor).Wait
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2200
		    main.registerBackup.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/backup.go:251
		    main.(*testRunner).runTest.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/test_runner.go:716
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor failure:
		  - error with attached stack trace:
		    main.(*monitor).wait.func2
		    	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/cluster.go:2245
		    runtime.goexit
		    	/usr/local/go/src/runtime/asm_amd64.s:1357
		  - monitor task failed:
		  - pq: importing 50 ranges: importing span /Table/55/{2-3}: fetching "523008157595336706.sst": open /mnt/data1/cockroach/extern/teamcity-1703424-1579679121-13-n3cpu4/full/523008157595336706.sst: no such file or directory

```

<details><summary>Repro</summary><p>


Artifacts: [/backupTPCC](https://teamcity.cockroachdb.com/viewLog.html?buildId=1703424&tab=artifacts#/backupTPCC)
[roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2AbackupTPCC.%2A&sort=title&restgroup=false&display=lastcommented+project)

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/c-deps/libroach/incremental_iterator.cc
+++ b/c-deps/libroach/incremental_iterator.cc
@@ -113,7 +113,7 @@ void DBIncrementalIterator::maybeSkipKeys() {
       // phantom MVCCKey.Keys. These keys may not be seen by the main iterator
       // due to aborted transactions and keys which have been subsumed due to
       // range tombstones. In this case we can SeekGE() the TBI to the main iterator.
-      DBKey seek_to;
+      DBKey seek_to = {};
       // NB: We don't ToDBKey as iter_key is already split.
       seek_to.key = ToDBSlice(iter_key);
       state = DBIterSeek(time_bound_iter.get(), seek_to);
@@ -134,7 +134,7 @@ void DBIncrementalIterator::maybeSkipKeys() {
       // keys. The main iterator is seeked to the TBI in hopes that many keys
       // were skipped. Note that a Seek() is an order of magnitude more
       // expensive than a Next().
-      DBKey seek_to;
+      DBKey seek_to = {};
       // NB: We don't ToDBKey as iter_key is already split.
       seek_to.key = ToDBSlice(tbi_key);
       state = DBIterSeek(iter.get(), seek_to);

--- a/pkg/ccl/backupccl/backup_planning.go
+++ b/pkg/ccl/backupccl/backup_planning.go
@@ -67,7 +67,7 @@ var fullClusterSystemTables = []string{
 var useTBI = settings.RegisterBoolSetting(
 	"kv.bulk_io_write.experimental_incremental_export_enabled",
 	"use experimental time-bound file filter when exporting in BACKUP",
-	false,
+	true,
 )
 
 var backupOptionExpectValues = map[string]sql.KVStringOptValidate{

